root/static/scripts/relationship-editor/common/entity.js

```diff
@@ -145,13 +147,14 @@ const RE = MB.relationshipEditor = MB.relationshipEditor || {};
 
                                 if (currentValue && !newValue) {
                                     _.each(group.values.slice(0), function (r) {
-                                        r.linkOrder(0);
+                                        r.linkOrder(0); 
```

Comment:
There's still extra whitespace being added at the end of the line here - please remove :)